MLS.Agent.Tools/RelativePath.cs

```diff
@@ -11,7 +11,7 @@ public abstract class RelativePath
     {
         private string _value;
 
-        protected RelativePath(string value)
+        protected internal RelativePath(string value)
```

Comment:
This change should be unnecessary.

Reply:
Removed internal. The idea was to avoid unknown subclasses of RelativePath, because we rely on knowing the subtypes, but since this is not an API assembly, I aggree it is unnecessary.